landuse_classification/Image_Classification.ipynb

```diff
@@ -63,7 +62,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
```

Comment:
These details should get cleared by the kernel stripping script.

Reply:
Fixed!